src/rooms/Game.ts

```
@@ -0,0 +1,117 @@
+      // allow disconnected client to reconnect into this room until 20 seconds
+      await this.allowReconnection(client, 20);
+
+      // client returned! let's re-activate it.
+      this.state.players.get(client.sessionId).connected = true;
+    } catch (e) {
+      // 20 seconds expired. let's remove the client. 
+      this.state.players.delete(client.sessionId);
+
+      // Reset the score  as a user already left the room
+      this.state.draws = 0;
+      this.state.players.forEach((player) => (player.won = 0));
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)